style.css

```diff
@@ -351,3 +351,97 @@ h2 {
   align-items: flex-start;
   gap: 15px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 70%;
+  background-position-x: right;
+  background-position-y: 20px;
+  border-radius: 90px 0 0 0;
+  padding-top: 103px;
+}
+
+.contact-title {
+  text-align: center;
+  font-size: 40px;
+  color: white;
+  margin-bottom: 20px;
+}
+
+.p-contact {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  padding: 24px;
+}
```

Comment:
- [ ] Kindy make sure to use to correct typography on this paragraph in order to meet the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/199470128-0182606a-c0e3-4b95-9935-605eb7286eea.png)

   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/199470213-ca3d726c-d24a-4376-923e-25ac3268af99.png)